src/components/reports/formSteps/Confirmation.tsx

```diff
@@ -10,11 +10,24 @@ interface Props {
 const Confirmation: FC<Props> = (props) => {
   const { formRef } = props;
 
-  const formObj = formRef.current.values;
+  let formObj = formRef.current.values;
+
+  // parse the variants and add them to the formObj as separate keys
+  if (formObj.variant && formObj.variant.length > 0) {
+    formObj.variant.map((variant: { [key: string]: string }, index: number) => {
+      formObj = { ...formObj, [`variant${index}`]: variant };
+    });
+    delete formObj.variant; // delete the old redundant variant key
+  }
```

Comment:
oopf yeah that is annoying that we have to do that isn't it. Agree that we should get this into users hands before we do any more on it - see how useful it is